zerver/views/home.py

```diff
@@ -75,7 +75,7 @@ def approximate_unread_count(user_profile):
         flags=UserMessage.flags.read).count()
 
 def sent_time_in_epoch_seconds(user_message):
-    # type: (UserMessage) -> Optional[float]
+    # type: (Optional[UserMessage]) -> Optional[float]
     # user_message is a UserMessage object.
     if not user_message:
```

Comment:
Definitely better, yes; also removed the comment beneath, which is superfluous since we have the annotation now! A case of gaining confidence of how much to put into commits - and how much to allow "unrelated" changes, eg. extra comments or code changes!